lib/policy_machine.rb

```diff
@@ -66,7 +66,16 @@ def add_association(user_attribute_pe, operation_set, object_attribute_pe)
   # object_or_attribute is an object or object attribute.
   #
   # TODO: add option to ignore policy classes to allow consumer to speed up this method.
+  # TODO: Parallelize the two component checks
   def is_privilege?(user_or_attribute, operation, object_or_attribute, options = {})
+    is_privilege_ignoring_prohibitions?(user_or_attribute, operation, object_or_attribute, options) &&
+      !is_privilege_ignoring_prohibitions?(user_or_attribute, PM::Prohibition.new(operation), object_or_attribute, options)
```

Comment:
I thought ```is_privilege_ignoring_prohibitions``` would raise if not passed an Operation, Symbol or String.  Is there some ```PM::Prohibition``` inheritance I'm missing somewhere?